run.sh

```diff
@@ -14,6 +14,10 @@
 
 # The docker container will compile our "testoutput" module as "test-runner"
 
+export GOLANG_VERSION=1.13.5
+export GOPATH=/go
+export PATH="$GOPATH/bin:/usr/local/go/bin:$PATH"
```

Comment:
I actually wonder whether just the PATH export would be sufficient, but I don't know if the other ENV vars are important.

Reply:
`GOPATH` definitely needs to be set otherwise it defaults to `~/go`.
I wouldn't know what we need the `GOLANG_VERSION` for. If it works without, I'd skip adding it.

Reply:
ok - will do. Appreciate the help!